### ISSUE TITLE
feat: Adding dark mode

### ISSUE THREAD

**Issue by VikramNagwal** [conventional-commits/conventionalcommits.org#607]
Providing a dark mode option for the website would offer users an additional choice for their comfort.

**Comment by damianopetrungaro** [comment]
@VikramNagwal feel free to propose a UI in this thread and if the community likes it we can move on with a PR as well 

We are open to such ideas!

**Comment by VikramNagwal** [comment]
I am presenting my proposed UI design for the website. Please provide your feedback on any additions or removals you think are necessary.
Figma Design : 
https://www.figma.com/design/KiYec7d0kvLQ8OfSZ3ArwC/conventionalCmmits(dark)?node-id=0-1&t=SlqEVi0pkk5vV5jJ-1

**Comment by damianopetrungaro** [comment]
I don't think this colours are accessible and easy to read, what about something like [this](https://dribbble.com/shots/22611851-Dimension-dev-Re-Design) color set?

**Comment by VikramNagwal** [comment]
It seems cool to use gradients for headlines, but what about subheadings? Overall, it's a good UI Design.

**Comment by R4M4NCHICK1** [comment]
I agree, subtitles and A dark interface should be

вт, 12 лист. 2024 р., 13:51 користувач Vikram ***@***.***>
пише:

**Comment by Jorgagu** [comment]
👋 Any update on this topic @vikramnagwal @damianopetrungaro ?

**Comment by damianopetrungaro** [comment]
If we end up having a readable color set and a PR, I am happy to review that.
I won't be able to code this myself as a lower priority tho, contribution are more than welcome!
